plugins/bulk-import/src/components/AddRepositories/AddRepositoriesPage.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import { Content, Header, Page } from '@backstage/core-components';
+
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import Accordion from '@mui/material/Accordion';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import Typography from '@mui/material/Typography';
+
+import { AddRepositoriesFormValues } from '../../types';
+import { AddRepositoriesForm } from './AddRepositoriesForm';
+import { Illustrations } from './Illustrations';
+
+export const AddRepositoriesPage = () => {
+  const initialValues: AddRepositoriesFormValues = {
+    repositoryType: 'repository',
+    repositories: [],
+    organizations: [],
+    approvalTool: 'git',
+  };
+
+  return (
+    <Page themeId="tool">
+      <Header title="Add repositories" type="Bulk import" typeLink=".." />
+      <Content>
+        <Accordion defaultExpanded>
+          <AccordionSummary
+            expandIcon={<ExpandMoreIcon />}
+            id="add-repository-summary"
+          >
+            <Typography variant="h5">
+              Add repositories to Red Hat Developer Hub in 5 steps
+            </Typography>
+          </AccordionSummary>
+          <AccordionDetails id="add-repository-illustrations">
```

Comment:
We can use `display: flex` in the Illustrations container and use `justify-content: space-around` instead of hard coded right padding of illustrations.
<img width="1285" alt="rhidp_1664_pr_review" src="https://github.com/janus-idp/backstage-plugins/assets/26255262/a0d9da85-5932-4962-a774-e9cc83db2a6e">

Reply:
Adding `display: flex` to the container wrapping the illustrations, the mobile view looks like this
<img width="1069" alt="Screenshot 2024-04-05 at 10 41 06 AM" src="https://github.com/janus-idp/backstage-plugins/assets/22490998/0ce60eb4-3b84-425c-ad7c-5bed551ec757">

Reply:
Oh I didn't know we need to deal with mobile view as well. In that case is adding a `overflow: scroll` acceptable? The current padding doesn't work on my laptop default screen. There might be other displays we need to review and get some layout design from UX team for better mobile screen display.

https://github.com/janus-idp/backstage-plugins/assets/26255262/0ef6706e-a585-4321-b51a-7a17b8d689da